src/consensus/pbft_manager.cpp

```diff
@@ -1435,14 +1434,14 @@ std::optional<vec_blk_t> PbftManager::comparePbftBlockScheduleWithDAGblocks_(std
     }
     cert_sync_block_.pbft_blk = std::move(pbft_block);
 
-    return std::move(dag_blocks_order);
+    return {std::move(dag_blocks_order)};
   }
   syncPbftChainFromPeers_(missing_dag_blk, anchor_hash);
   return {};
 }
 
 bool PbftManager::pushCertVotedPbftBlockIntoChain_(taraxa::blk_hash_t const &cert_voted_block_hash,
-                                                   std::vector<std::shared_ptr<Vote>> const &cert_votes_for_round) {
+                                                   std::vector<std::shared_ptr<Vote>> &&cert_votes_for_round) {
```

Comment:
I don't understand here. &&cert_votes_for_round is RValue, and std::move is for LValue. But you use "std::move(cert_votes_for_round)" is std::move for a RValue here. Could you explain that? Thanks

Reply:
https://stackoverflow.com/questions/31213539/c-stdmove-with-rvalue-reference-is-not-moving-contents